regexp/regexp_test.go

```diff
@@ -0,0 +1,125 @@
+package regexp
+
+import (
+	"github.com/stretchr/testify/require"
+	"regexp"
+	"runtime"
+	"strconv"
+	"sync"
+	"testing"
```

Comment:
Can you separate the imports into two groups, like we normally do?

Reply:
🤦 I really need to get my IDE to do it right for me. Fixed in 9c844f2ea1502369964e908ca7af8c4e476aeeac